examples/dev/demo_shape_creation.py

```diff
@@ -34,77 +34,77 @@ def time_me(label, func):
     return res
 
 
-if __name__ == '__main__':
-
-    parser = argparse.ArgumentParser(description='')
+parser = argparse.ArgumentParser(description='')
+
+parser.add_argument(
+    '-n',
+    '--n_polys',
+    type=int,
+    default=5000,
+    help='number of polygons to show',
+)
+parser.add_argument(
+    '-t',
+    '--type',
+    type=str,
+    default='path',
+    choices=['path', 'path_concat', 'polygon', 'rectangle', 'ellipse'],
+)
+parser.add_argument(
+    '-c',
+    '--concat',
+    action='store_true',
+    help='concatenate all coordinates to a single mesh',
+)
+parser.add_argument(
+    '-v', '--view', action='store_true', help='show napari viewer'
+)
+parser.add_argument(
+    '--properties', action='store_true', help='add dummy shape properties'
+)
+
+args = parser.parse_args()
```

Comment:
This is bad change. As it trigger always parsing content of `sys.argv` and may lead to crash of test. I think that we should use `init_globals` argument of `run_path` to detect such use. maybe something like:
```suggestion
if locals().get("test_run", False):
    args = parser.parse_args([])
else:
    args = parser.parse_args([])
```